controllers/api/v1/health_check_test.go

```diff
@@ -0,0 +1,41 @@
+package apiv1
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	. "google-scraper/tests"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("HealthCheckController", func() {
+	Describe("GET /api/v1/health-check", func() {
+		It("returns 200 status code", func() {
+			response := MakeRequest("GET", "/api/v1/health-check", nil)
+
+			Expect(response.StatusCode).To(Equal(http.StatusOK))
+		})
+
+		It("returns correct JSON response", func() {
+			expectedResponse := `{
+				"data": {
```

Comment:
@junan Can you check with Hoang's PR?
He did with checking valid json scheme

Reply:
I have seen his PR where he used schema matcher. But I prefer to use Gomega's `MatchJSON` matcher  😄